lib/bureaucrat/markdown_writer.ex

```diff
@@ -174,7 +191,7 @@ defmodule Bureaucrat.MarkdownWriter do
   defp group_records(records) do
     by_controller = Enum.group_by(records, &get_controller/1)
```

Comment:
Sounds good!

The controllers are normally in separate files. Currently, the pre-ordering is only by line, which is pretty useless when looking at several files. But if we order the __records by file path and line__ in the first place, and use `stable_group_by` here, the output from `web/channels/*` would come before `web/controllers/*` which would make sense. I'm just not sure how to sort by two fields; probably just make a tuple `{filepath, line}`.

I'm gonna try this and add it to the PR.